server/src/instant/dash/routes.clj

```diff
@@ -943,6 +943,14 @@
 ;; ---------------
 ;; Email templates
 
+(defn sender-verification-get [req]
+  (let [{{app-id :id} :app} (req->app-and-user! :admin req)
+        {postmark-id :postmark_id}
+        (app-email-template-model/get-by-app-id-and-email-type
+         {:app-id app-id :email-type "magic-code"})]
+    (response/ok {:verification (when postmark-id
+                                  (:body (postmark/get-sender! {:id postmark-id})))})))
```

Comment:
Might be worth doing a select-keys here in case there is stuff in the payload we don't want to expose to the user.